### ISSUE TITLE
multi emails

### ISSUE THREAD

**Issue by oscaroxy** [RobinHerbots/Inputmask#2538]
Hi, is there a way to insert more than one email in inputmask email?
thanks

**Comment by oscaroxy** [comment]
I should change the validator, but I don't know from how to how to change it.... thanks

**Comment by RobinHerbots** [comment]
Hi @oscaroxy,
I didn't test it but by updating the mask.

```
Inputmask({
    alias: "email",
    mask: "*{1,64}[.*{1,64}][.*{1,64}][.*{1,63}]@-{1,63}.-{1,63}[.-{1,63}][.-{1,63}](, *{1,64}[.*{1,64}][.*{1,64}][.*{1,63}]@-{1,63}.-{1,63}[.-{1,63}][.-{1,63}]){*}",
```

This should allow  test@test.com, test2@test.com, ....

**Comment by RobinHerbots** [comment]
#2557, #2558

**Comment by oscaroxy** [comment]
unfortunately, don't work, I can write only:

pippo@pippo.it, pluto

I can't write other "@"

thanks

**Comment by RobinHerbots** [comment]
#2559

**Comment by oscaroxy** [comment]
I didn't understand your reply, the issue #2559 bring here.
the inputmask with your update 

`mask: "*{1,64}[.*{1,64}][.*{1,64}][.*{1,63}]@-{1,63}.-{1,63}[.-{1,63}][.-{1,63}](, *{1,64}[.*{1,64}][.*{1,64}][.*{1,63}]@-{1,63}.-{1,63}[.-{1,63}][.-{1,63}]){*}"`

don't works, in fact if I try to insert more one email, the inputmask don't allow to insert more then one "@" simbol, so I can write at most "pippo@walt.it, pluto" without add again "@", like this "pippo@walt.it, pluto@walt.it"

thanks

**Comment by oscaroxy** [comment]
nothing?

**Comment by oscaroxy** [comment]
I tried other combination of mask, but I don't obtain that I want... the "@" doesn't appears.  can you help me?

**Comment by RobinHerbots** [comment]
@oscaroxy ,

Yes I will/am working on extending the email mask to allow multiple entries.

**Comment by oscaroxy** [comment]
thanks, I'll wait your work, your library is the top for me.

**Comment by RobinHerbots** [comment]
There is some progress.
Add the separator option to the email mask and enable entering 2 emails.  

```
  Inputmask("email", {
            separator: ", "
        }).mask("test2");
```

todo: 
- check why the email template is not shown for the second email.
- enable the email mask in combi with the quantifier syntax.

**Comment by oscaroxy** [comment]
I tried but it doesn't work, it work like before, that is:

pippo@pluto.it, paperino

and I can't write the "@" 
thanks

**Comment by RobinHerbots** [comment]
Did you use the latest from the dist folder ?

**Comment by oscaroxy** [comment]
yes, with the last advice :

`mask: "*{1,64}[.*{1,64}][.*{1,64}][.*{1,63}]@-{1,63}.-{1,63}[.-{1,63}][.-{1,63}](, *{1,64}[.*{1,64}][.*{1,64}][.*{1,63}]@-{1,63}.-{1,63}[.-{1,63}][.-{1,63}]){*}"`

**Comment by oscaroxy** [comment]
I'm sorry, I didn't know of the new release 5.0.7-beta.32, I had .24, now I downloaded the new release and the inputText work fine for 2 email, but not for more email. How can I do for more email?
thanks

**Comment by RobinHerbots** [comment]
@oscaroxy ,

Use the latest version 5.0.7-beta.34  ;-)

And have a try with

```
  Inputmask("email", {
            separator: ", "
            quantifier: 5
        }).mask("test2");

```

This will allow 6 e-mail adresses.

**Comment by oscaroxy** [comment]
I'm sorry for delay, I tried your update and it work, 6 email are enough for now :-D 
thanks a lot, your library is very very nice

**Comment by RobinHerbots** [comment]
Resolved. Multi-address entry was enabled by the nested-optionals / masklength recalculation work (commits e700c17f + 63ae6036, 2021), and the reporter confirmed it works ("I tried your update and it work, 6 email are enough for now", 2021-12-30). Closing as answered.
